packages/e2e-playwright/tests/ui/card/binLookup/plcc/plcc.noluhn.nodate.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { test, expect } from '../../../../../fixtures/card.fixture';
+import { getStoryUrl } from '../../../../utils/getStoryUrl';
+import { PLCC_NO_LUHN_NO_DATE_WOULD_FAIL_LUHN, TEST_CVC_VALUE } from '../../../../utils/constants';
+import { URL_MAP } from '../../../../../fixtures/URL_MAP';
+
+test.describe('Testing binLookup endpoint for a response that should indicate a luhn check is not required)', () => {
+    test('Test a PLCC card, that does not require a date, becomes valid with a number that fails the luhn check', async ({ card, page }) => {
+        //
+        const componentConfig = { brands: ['mc', 'visa', 'amex', 'bcmc', 'synchrony_plcc'] };
+
+        await card.goto(getStoryUrl({ baseUrl: URL_MAP.card, componentConfig }));
+
+        await card.isComponentVisible();
+
+        // Number that identifies as plcc but fails luhn
+        await card.typeCardNumber(PLCC_NO_LUHN_NO_DATE_WOULD_FAIL_LUHN);
+
+        // Confirm plcc brand
+        let brandingIconSrc = await card.brandingIcon.getAttribute('src');
+        expect(brandingIconSrc).toContain('synchrony_plcc.svg');
+
+        // Confirm date is hidden
+        await expect(card.expiryDateField).not.toBeVisible();
+
+        // Fill cvc
+        await card.typeCvc(TEST_CVC_VALUE);
+
+        // PM is valid
+        let cardValid = await page.evaluate('window.component.isValid');
+        expect(cardValid).toEqual(true);
+
+        // Delete number
+        await card.deleteCardNumber();
+
+        // Allow time for icon to load
+        await page.waitForTimeout(500);
+
+        // UI reset
+        brandingIconSrc = await card.brandingIcon.getAttribute('src');
+        expect(brandingIconSrc).toContain('nocard.svg');
+
+        // PM is not valid
+        cardValid = await page.evaluate('window.component.isValid');
+        expect(cardValid).toEqual(false);
```

Comment:
Is there a way to test this without checking the 'internals' of the component state? Maybe by attempting to make the payment and then verifying that there is an error in the UI?

Reply:
I don't know, probably. 
But since I'm still testing internal state on line 30, to check that the component _is_ valid (and I don't want to press the Pay button then 'cos I want todo more in the UI) - I'll leave it, as it is for now,